reference/confirm/models/alert.v1.yaml

```diff
@@ -0,0 +1,58 @@
+# Do we link to google docs here?
+# https://docs.google.com/document/d/1zpKhSEqZMyKXoY_3sADTtOpQ-4v1AOZ4qIuZh3efU1U
+type: object
+title: An alert configuration
+description: Configuration for an alert notification.
+properties:
+  enabled:
+    type: boolean
+  threshold:
+    # Question: Should there be a mg/dL unit defined in the schema? Since we
```

Comment:
Not sure I understand the question completely, but I agree with having a standardized unit that the api expects values to be in, rather than something configurable.

Reply:
How does type safety play a role here?

Reply:
So right now, what's specified, is that an integer is sent. This integer is understood to be defined in units of mg/dL. There is nothing enforcing that, or able to double check that however.

I'm wondering if it's worthwhile to specify the mg/dL more explicitly. So for example, if we changed it from an integer to something like:
```json
threshold: { value: 100, units: "mg/dL" }```
Which would allow us to error if the units value wasn't supported. We need only support the units we want, it's just clearer what those units are, and it fails if someone sends a different unit, which could prevent mistakes by 3rd parties...

Which kinda leads to the reason why we might not care about this... If we don't expect 3rd parties to use the API, then there's very little value to this... If we're only ever talking to ourselves, and/or trusted partners, we can communicate the units to them, and trust that they'll send the correct units. In that case, adding this would only add some size to the requests. But that said, I feel like it might be worth it, given that it's an important thing to be explicit about.

Reply:
> How does type safety play a role here?

I'm bending the term a little bit... it's not exactly type safety, in this case it's ensuring "unit safety".

Reply:
We could even specify something like a string that ends with "mg/dL", via a regex match or something. The point is to be very explicit about the units we're expecting, and to allow an error to be generated if something else is sent. With a bare integer, we have no way to do that.